co-op/index.html

```diff
@@ -58,8 +58,8 @@
                             </a>
                             {% endif %}
                         </div>
-                        <div class="content">
-                            <a id="{{ item.identification }}" class="header">{{ item.title }}</a>
+                        <div class="company-card-footer content">
+                            <a class="company-card-footer-link header" id="{{ item.identification }}">{{ item.title }}</a>
```

Comment:
I recommend refactoring this `a`nchor to a `span`, because the footer isn't/is no longer serving as a link to any other page.  It also looks like you only need the `header` class to achieve the same styling :)

Reply:
Good point, I'll do that!